ANSI escape sequences are not very convenient in log files...
env_logger allows to disable them via env. variable and also prints timestamp.

--- a/src/garage/main.rs
+++ b/src/garage/main.rs
@@ -59,7 +59,7 @@
 	if std::env::var("RUST_LOG").is_err() {
 		std::env::set_var("RUST_LOG", "netapp=info,garage=info")
 	}
-	pretty_env_logger::init();
+	env_logger::init();
 	sodiumoxide::init().expect("Unable to init sodiumoxide");
 
 	let opt = Opt::from_args();
--- a/src/garage/Cargo.toml
+++ b/src/garage/Cargo.toml
@@ -34,1 +34,1 @@
-pretty_env_logger = "0.4"
+env_logger = "0.8"
